localstack_client/config.py

```diff
@@ -9,122 +8,122 @@
-    "cognito-idp": "{proto}://{host}:4590",
-    "cognito-identity": "{proto}://{host}:4591",
-    "sts": "{proto}://{host}:4592",
-    "iam": "{proto}://{host}:4593",
-    "rds": "{proto}://{host}:4594",
-    "rds-data": "{proto}://{host}:4594",
-    "cloudsearch": "{proto}://{host}:4595",
-    "swf": "{proto}://{host}:4596",
-    "ec2": "{proto}://{host}:4597",
-    "elasticache": "{proto}://{host}:4598",
-    "kms": "{proto}://{host}:4599",
-    "emr": "{proto}://{host}:4600",
-    "ecs": "{proto}://{host}:4601",
-    "eks": "{proto}://{host}:4602",
-    "xray": "{proto}://{host}:4603",
-    "elasticbeanstalk": "{proto}://{host}:4604",
-    "appsync": "{proto}://{host}:4605",
-    "cloudfront": "{proto}://{host}:4606",
-    "athena": "{proto}://{host}:4607",
-    "glue": "{proto}://{host}:4608",
-    "sagemaker": "{proto}://{host}:4609",
-    "sagemaker-runtime": "{proto}://{host}:4609",
-    "ecr": "{proto}://{host}:4610",
-    "qldb": "{proto}://{host}:4611",
-    "qldb-session": "{proto}://{host}:4611",
-    "cloudtrail": "{proto}://{host}:4612",
-    "glacier": "{proto}://{host}:4613",
-    "batch": "{proto}://{host}:4614",
-    "organizations": "{proto}://{host}:4615",
-    "autoscaling": "{proto}://{host}:4616",
-    "mediastore": "{proto}://{host}:4617",
-    "mediastore-data": "{proto}://{host}:4617",
-    "transfer": "{proto}://{host}:4618",
-    "acm": "{proto}://{host}:4619",
-    "codecommit": "{proto}://{host}:4620",
-    "kinesisanalytics": "{proto}://{host}:4621",
-    "kinesisanalyticsv2": "{proto}://{host}:4621",
-    "amplify": "{proto}://{host}:4622",
-    "application-autoscaling": "{proto}://{host}:4623",
-    "kafka": "{proto}://{host}:4624",
-    "apigatewaymanagementapi": "{proto}://{host}:4625",
-    "timestream": "{proto}://{host}:4626",
-    "timestream-query": "{proto}://{host}:4626",
-    "timestream-write": "{proto}://{host}:4626",
-    "s3control": "{proto}://{host}:4627",
-    "elbv2": "{proto}://{host}:4628",
-    "support": "{proto}://{host}:4629",
-    "neptune": "{proto}://{host}:4594",
-    "docdb": "{proto}://{host}:4594",
-    "servicediscovery": "{proto}://{host}:4630",
-    "serverlessrepo": "{proto}://{host}:4631",
-    "appconfig": "{proto}://{host}:4632",
-    "ce": "{proto}://{host}:4633",
-    "mediaconvert": "{proto}://{host}:4634",
-    "resourcegroupstaggingapi": "{proto}://{host}:4635",
-    "resource-groups": "{proto}://{host}:4636",
-    "efs": "{proto}://{host}:4637",
-    "backup": "{proto}://{host}:4638",
-    "lakeformation": "{proto}://{host}:4639",
-    "waf": "{proto}://{host}:4640",
-    "wafv2": "{proto}://{host}:4640",
-    "config": "{proto}://{host}:4641",
-    "configservice": "{proto}://{host}:4641",
-    "mwaa": "{proto}://{host}:4642",
-    "fis": "{proto}://{host}:4643",
-    "meteringmarketplace": "{proto}://{host}:4644",
-    "transcribe": "{proto}://{host}:4566",
-    "mq": "{proto}://{host}:4566"
+_service_ports: Dict[str, int] = {
```

Comment:
I'm inclined to remove this mapping altogether, as it has been painful and is no longer in use. This dates back to the [`0.11.x` era](https://github.com/localstack/localstack/issues/2983) (before we even introduced the edge port :) ), and is not working anymore anyways. v2 could be a good opportunity to get rid of `USE_LEGACY_PORTS`! 🧹

Reply:
I'm not 100% sure what that will do to the controll paths of legacy s3 and lambda providers. We may still access them in some adapter code but not 100% sure